signingscript/src/signingscript/vendored/mozbuild/mozbuild/mozinfo.py

```diff
@@ -75,6 +75,7 @@ def build_dict(config, env=os.environ):
     d['debug'] = substs.get('MOZ_DEBUG') == '1'
     d['nightly_build'] = substs.get('NIGHTLY_BUILD') == '1'
     d['release_or_beta'] = substs.get('RELEASE_OR_BETA') == '1'
+    # XXX pinebuild?
```

Comment:
can you file an issue for this?

Reply:
https://mozilla-hub.atlassian.net/browse/RELENG-705